File: src/第二章/类和属性.java
package 第二章;

/**
 * 日期 : 2020/10/17.
 * 创建 : xin.li
 * 描述 :
 */
public class 类和属性 {
    public static void main(String[] args) {
        Person2 person2 = new Person2("李鑫" , 28 , false);
        System.out.println("name = " + person2.getName());
        System.out.println("age = " + person2.getAge());
        System.out.println("isMarried = " + person2.isMarried());
    }
}

File: src/第二章/类和属性.kt
package 第二章

fun main(args: Array<String>) {
    val person2 = Person2("李三" , 19 , true);
    println(person2.name)
    println(person2.age)
    println(person2.isMarried)
    person2.isMarried = false;
    println(person2.isMarried)
}

/**
 * 日期 : 2020/10/17.
 * 创建 : xin.li
 * 描述 : 值对象
 */
class Person(val name : String)

/**
 * bean属性
 */
class Person2(
    val name : String,
    var age : Int,
    var isMarried : Boolean
)

